### ISSUE TITLE
Not working on mint 17

### ISSUE THREAD

**Issue by rafaelribeiroo** [GabMus/HydraPaper#32]
When I try to change my desktop, show this message. Anyone can help me?

Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Gb9ahe0pkE: Connection refused
(hydrapaper:2): Gtk-WARNING **: 00:03:12.568: Theme parsing error: gtk.css:3415:30: The style property GtkButton:image-spacing is deprecated and shouldn't be used anymore. It will be removed in a future version
(hydrapaper:2): dconf-WARNING **: 00:04:42.923: failed to commit changes to dconf: Não foi possível conectar: Connection refused

**Comment by GabMus** [comment]
Not sure what those errors mean, but HydraPaper never officially supported mint. Possibly I'd need to edit the dconf entry for cinnamon instead of the GNOME one, but I should install mint to know what it's called.

Possibly I will try in a VM at some point.
